video/out/wayland_common.c

```diff
@@ -957,7 +957,8 @@ static void surface_handle_enter(void *data, struct wl_surface *wl_surface,
         update_output_geometry(wl, old_geometry, old_output_geometry);
 
     MP_VERBOSE(wl, "Surface entered output %s %s (0x%x), scale = %f, refresh rate = %f Hz\n",
```

Comment:
How about we move it inside the loop instead?

Reply:
I had that thought too but `update_output_geometry` probably sets `wl->scaling`

Reply:
Yep.